package-lock.json

```diff
@@ -2867,25 +2867,29 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "bundled": true,
+          "resolved": false,
+          "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
```

Comment:
why all these changes in this file?

Reply:
package-lock.json is created and maintained by npm nothing we should manage manually